ddht/v5_1/alexandria/messages.py

```diff
@@ -79,6 +89,34 @@ class PongMessage(AlexandriaMessage[PongPayload]):
     payload: PongPayload
 
 
+@register
+class FindNodesMessage(AlexandriaMessage[FindNodesPayload]):
```

Comment:
I know this isn't implementing v51 so the names don't have to match, but it seems extra confusing that `FindNodeMessage` is the v51 message and `FindNodesMessage` is the alexandria message, I can see myself mixing those two up a lot. I think calling this `FindNodeMessage` might be easier on future us.

Reply:
I'd rather rename the v5 one because 1) we're likely to fully remove the v5 implementation at some point and 2) at least in v5.1, the plural feels more correct since in it's typical usage it finds multiple nodes.

Reply:
Okay, I'm also happy with the plural `FindNodesMessage`! I just expect it'll be easier on us if whichever name this uses it has the same name in every protocol.